### ISSUE TITLE
extended keyboard input is not working on Android

### ISSUE THREAD

**Issue by avpanov** [bk138/multivnc#129]
If you'd like to put out an incentive for fixing this bug, you can do so at https://issuehunt.io/r/bk138/multivnc?tab=idle

**Is your bug report about the Desktop Multivnc or the Mobile MultiVNC?**
 - Desktop (wxWidgets) or Mobile (Android):
Mobile

**Which MultiVNC version are you using?**

1.9.10
**Describe the bug**

I use MultiVNC with recent termux installation with tigervnc on Huawei Mediapad T5 tablet (based on Android 8.0). When I switch the keyboard either virtual or physical (ordinary USB keyboard) to Russian Layout, nothing is typeset in MultiVNC. It is possible to typeset Russian keys inside  termux terminal. The output from xev with USB keyboard is below:

-----------------

KeyRelease event, serial 35, synthetic NO, window 0x1a00001,
    root 0x4f, subw 0x0, time 333038650, (-186,72), root:(685,615),
    state 0x0, keycode 24 (keysym 0x71, q), same_screen YES,
    XLookupString gives 1 bytes: (71) "q"
    XFilterEvent returns: False

KeyPress event, serial 35, synthetic NO, window 0x1a00001,
    root 0x4f, subw 0x0, time 333042308, (-186,72), root:(685,615),
    state 0x0, keycode 247 (keysym 0x439, (no name)), same_screen YES,
    XLookupString gives 0 bytes: 
    XFilterEvent returns: False

KeyRelease event, serial 35, synthetic NO, window 0x1a00001,
    root 0x4f, subw 0x0, time 333042308, (-186,72), root:(685,615),
    state 0x0, keycode 247 (keysym 0x439, (no name)), same_screen YES,
    XLookupString gives 0 bytes: 
    XFilterEvent returns: False

KeyPress event, serial 35, synthetic NO, window 0x1a00001,
    root 0x4f, subw 0x0, time 333042709, (-186,72), root:(685,615),
    state 0x0, keycode 230 (keysym 0x446, (no name)), same_screen YES,
    XLookupString gives 0 bytes: 
    XFilterEvent returns: False

-----------------

**For the Mobile Version (please complete the following information):**
 - Android version: 8.0.0
 - Installed from Play Store, F-Droid, Amazon Appstore:
 - F-Droid

**Additional context**

**Comment by bk138** [comment]
Thank you very much for this nice report! Did this work with previous versions of MultiVNC?

**Comment by avpanov** [comment]
Also, I have tried 1.8.12 version from Google play with the same result.

**Comment by bk138** [comment]
OK, so this probabyl never worked. is this possible to reproduce with https://github.com/AnySoftKeyboard/AnySoftKeyboard ? I.e. install russian keyboard, hit р and nothing appears?

**Comment by avpanov** [comment]
Yes, you can install Russian keyboard for Anysoft, switch to it to try. Both virtual and physical keyboards show similar behavior with xev. Open and Simple keyboards from F-droid have the similar behavior. 0x439 etc. are unicode codes for Russian letters.

**Comment by bk138** [comment]
Alright, will have a look. Might take a while though.
